src/BCEngineMCMC.cxx

```diff
@@ -1068,15 +1130,20 @@ bool BCEngineMCMC::ParameterTreeMatchesModel(TTree* partree, bool checkObservabl
             return false;
         }
         if (GetParameter(i).GetLowerLimit() != p_lowerlimit)
-            BCLog::OutWarning(Form("BCEngineMCMC::ParameterTreeMatchesModel : Lower limit of parameter \"%s\" does not match.", p_name));
+            BCLog::OutDetail(Form("BCEngineMCMC::ParameterTreeMatchesModel : Lower limit of parameter \"%s\" does not match.", p_name));
         if (GetParameter(i).GetUpperLimit() != p_upperlimit)
-            BCLog::OutWarning(Form("BCEngineMCMC::ParameterTreeMatchesModel : Upper limit of parameter \"%s\" does not match.", p_name));
+            BCLog::OutDetail(Form("BCEngineMCMC::ParameterTreeMatchesModel : Upper limit of parameter \"%s\" does not match.", p_name));
```

Comment:
That was my original reasoning. But to suppress the warning, you'd have to put the model in the same state as the file is in before loading it. But the functionality of loading the file was implemented precisely to put a model into this state. (I believe this will be the most common use.) So I think it's best given as a detail and not a warning.

Reply:
I see, fine with me